Mathlib/MeasureTheory/Measure/Dirac.lean

```diff
@@ -5,6 +5,7 @@ Authors: Johannes Hölzl
 -/
 import Mathlib.MeasureTheory.Measure.Typeclasses
 import Mathlib.MeasureTheory.Measure.MutuallySingular
+import Mathlib.MeasureTheory.MeasurableSpace.CountablyGenerated
```

Comment:
This is an added import, to be able to state `dirac_ne_dirac` in the natural generality, i.e., assuming `MeasurableSpace.SeparatesPoints`.

If the import creep is unacceptable, I see the options:
 * move the general `dirac_ne_dirac` to a different (later) file than `Dirac.lean` (one option is the new `DiracProba.lean`), and possibly state a version here with suboptimal hypotheses that do not require new imports, such as `MeasurableSingletonClass`;
 * move `MeasurableSpace.SeparatesPoints` out of `CountablyGenerated.lean` (in fact its current placement does not appear very natural to me, would it be better to have a separate "typeclasses for measurable spaces" file with `MeasurableSingletonClass`, `SeparatesPoints`, etc?).

Reply:
I think this added import is fine: you are not adding much, and indeed you want to be able to use SeparatesPoints because that's the right generality.

I also agree that SeparatesPoints could be moved to another file.